src/Network/Network/help/New-AzLoadBalancerBackendAddressConfig.md

```diff
@@ -0,0 +1,137 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# New-AzLoadBalancerBackendAddressConfig
+
+## SYNOPSIS
+Returns a load balancer address config. 
+
+## SYNTAX
+
+```
+New-AzLoadBalancerBackendAddressConfig -IpAddress <String> -Name <String> -VirtualNetwork <PSVirtualNetwork>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Returns a load balancer address config. 
+
+## EXAMPLES
+
+### Example 1
+### New loadbalancer address config with virtual network reference
+```powershell
+PS C:\> $virtualNetwork = Get-AzVirtualNetwork -Name $vnetName -ResourceGroupName $resourceGroup
+
+New-AzLoadBalancerBackendAddressConfig -IpAddress "10.0.0.5" -Name "TestVNetRef" -VirtualNetwork $virtualNetwork
+
+
+Name                    : TestVNetRef
+IpAddress               : 10.0.0.5
+BackendIpConfigurations : null
+VirtualNetwork          : {
```

Comment:
why not only ID ?

Reply:
I updated it to show as a child resource, Name, ID to make it easier to read, it still will have the virtual network object as that is needed in the SDK but will look like the below in the console

New-AzLoadBalancerBackendAddressConfig -IpAddress "10.0.0.5" -Name "TestVNetRef" -VirtualNetwork $virtualNetwork

 


Name                            : TestVNetRef
IpAddress                       : 10.0.0.5
NetworkInterfaceIpConfiguration : null
VirtualNetwork                  : {
                                    "Name": "alegal-test-slb-api",
                                    "Etag": "W/\"606bb0ea-ee56-4d24-b75c-a1c496beb165\"",
                                    "Id": "/subscriptions/9b8218f9-902a-4d20-a65c-e98acec5362f/resourceGroups/alegal-test-2/providers/Microsoft.Network/virtualNetworks/alegal-test-slb-api"
                                  }